### DISCUSSION TITLE
Share clipboard between devices

### DISCUSSION THREAD

**Discussion by asas1asas200** [debauchee/barrier#1312] [Ideas]
If I can use it like dual monitor on one computer, there will be awesome.
Sometimes I use a computer to search information then I want to copy some content to another computer.
Can we make a text-only shared clipboard?

**Comment by asas1asas200** [comment]
Oh it's already can do it now, sorry for my fault.

**Reply by asas1asas200** [reply]
Can close this discussions.

**Reply by shymega** [reply]
Alright, I've also made an issue for this, which is now closed.
